packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -214,6 +219,11 @@ export default class QuanticSearchBox extends LightningElement {
       this.recentQueriesList.executeRecentQuery(
         this.recentQueries.indexOf(value)
       );
+      this.engine.dispatch(
+        this.actions.clearQuerySuggest({
+          id: this.searchBox.state.querySuggestionId,
```

Comment:
You already have the state in `this.state`
```suggestion
          id: this.state.querySuggestionId,
```